rtl_433_mqtt_autodiscovery/run.sh

```diff
@@ -51,5 +51,9 @@ else
   fi
 fi
 
+if [ ! -z ${MQTT_PORT+x} ]; then
+  MQTT_PORT="1883"
```

Comment:
Makes sense. When running in HAOS, the default is set at https://github.com/pbkhrv/rtl_433-hass-addons/blob/main/rtl_433_mqtt_autodiscovery/config.json#L14.

Let's remind ourselves of this?

```suggestion
# Set a default port for when the container is being run directly.
if [ ! -z ${MQTT_PORT+x} ]; then
  MQTT_PORT="1883"
```